cli-plugins/apps/command-reference/tanzu-apps-workload-apply.md

```diff
@@ -21,35 +22,39 @@ tanzu apps workload apply [name] [flags]
 
 ```console
 tanzu apps workload apply --file workload.yaml
+tanzu apps workload apply my-workload --image "-" --param-yaml maven=$"artifactId:hello-world\ntype: jar\nversion: 0.0.1\ngroupId: carto.run"
```

Comment:
I think we can/should remove the  --image "-" since we're removing source validation